src/game_state/Level.h

```diff
@@ -0,0 +1,24 @@
+#ifndef LEVEL_H_
+#define LEVEL_H_
+
+#include <memory>
+#include <SFML/Audio.hpp>
+
+#include "GameObject.h"
+#include "Platform.h"
+
+class Level : public GameObject {
+ public:
+  Level(std::shared_ptr<sf::Music> music,
+        std::shared_ptr<std::vector<Platform>> level);
+  ~Level();
+
+  std::shared_ptr<sf::Music> getMusic() { return music; }
+  std::shared_ptr<std::vector<Platform>> getLevel() { return level; }
+
+ private:
+  std::shared_ptr<sf::Music> music;
+  std::shared_ptr<std::vector<Platform>> level;
```

Comment:
right the level should contain only things that are put there by the level generator and it should be ordered such that it is easy for the GameRenderer to tell what is in the current view based on what the current elapsed time is. I added some comments explaining this